smartmontools.yaml

```
@@ -0,0 +1,74 @@
+package:
+  name: smartmontools
+  version: "7.4"
+  epoch: 0
+  description: Control and monitor S.M.A.R.T. enabled hard drives
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libcap-ng-dev
+      - linux-headers
+      - systemd-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: smartmontools-tag-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/smartmontools/smartmontools
+      tag: RELEASE_${{vars.smartmontools-tag-version}}
+      expected-commit: 0c0bada5eedb759c9d7bb50ff54959cf0809c5a9
+
+  - working-directory: smartmontools
+    uses: autoconf/configure
+    with:
+      opts: |
+        --sbindir=/usr/bin
+
+  - working-directory: smartmontools
+    uses: autoconf/make
+
+  - working-directory: smartmontools
+    uses: autoconf/make-install
```

```suggestion
  - working-directory: smartmontools
    pipeline:
    - uses: autoconf/configure
      with:
        opts: |
          --sbindir=/usr/bin
    - uses: autoconf/make
    - uses: autoconf/make-install
```
we can consider something like this to avoid duplication to working-directory.

---

nice - I was not aware that you could do this!

Added to the PR.